### ISSUE TITLE
Code the Loss Function 

### ISSUE THREAD

**Issue by REGATTE** [REGATTE/RobustNet#1]
Code the loss function in PyTorch as a class, enabling it be used in all of the models to be trained and evaluated later.
